ui/src/setting/SettingsPage.tsx

```diff
@@ -120,6 +120,34 @@ export const SettingsPage: React.FC = () => {
                     ))}
                 </Select>
             </FormControl>
+            <FormControl margin={'normal'} fullWidth>
+                <InputLabel>Datetime input style</InputLabel>
+                <Select
+                    fullWidth
+                    value={settings.datetimeInputStyle}
+                    onChange={(e) => {
+                        setSettings({
+                            variables: {
+                                settings: {
+                                    ...settings,
+                                    datetimeInputStyle: e.target.value as DatetimeInputStyle,
+                                },
+                            },
+                        })
+                            .then(() =>
+                                enqueueSnackbar('preferred datetime input style changed', {
```

Comment:
Omit the "preferred" in the two snackbar messages. A "preference" sounds like something that will be taken into account but not 100% used. In this case it _sets_ the date time style.